src/modules/member/entities/member-skill-mapping.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { MemberSkillMapping } from '@prisma/client';
+import { IdResponseType } from '@src/types/id-response-type';
+
+export class MemberSkillMappingEntity
+  extends IdResponseType
```

Comment:
```suggestion
    extends IntersectionType(
    IdResponseType,
    PickType(DateResponseType, ['createdAt']),
  )
```
기존에 만들어 놓으신 DateResponseType을 상속 받지 않으신 이유가 있으신가요?

Reply:
이거 전에 타입에러 나서 습관적으로 상속받지 않았는데 다시 확인해보니 에디터 에러였나보네요 수정하도록 하겠습니다.